cmd/cliflags/flags.go

```diff
@@ -1,36 +1,44 @@
 package cliflags
 
 const (
-	BaseURIDefault = "https://app.launchdarkly.com"
+	BaseURIDefault      = "https://app.launchdarkly.com"
+	DevStreamURIDefault = "https://stream.launchdarkly.com"
+	PortDefault         = "8765"
 
-	AccessTokenFlag = "access-token"
-	AnalyticsOptOut = "analytics-opt-out"
-	BaseURIFlag     = "base-uri"
-	DataFlag        = "data"
-	EmailsFlag      = "emails"
-	EnvironmentFlag = "environment"
-	FlagFlag        = "flag"
-	OutputFlag      = "output"
-	ProjectFlag     = "project"
-	RoleFlag        = "role"
+	AccessTokenFlag  = "access-token"
+	AnalyticsOptOut  = "analytics-opt-out"
+	BaseURIFlag      = "base-uri"
+	DataFlag         = "data"
+	DevStreamURIFlag = "dev-stream-uri"
+	EmailsFlag       = "emails"
+	EnvironmentFlag  = "environment"
+	FlagFlag         = "flag"
+	OutputFlag       = "output"
+	PortFlag         = "port"
+	ProjectFlag      = "project"
+	RoleFlag         = "role"
 
 	AccessTokenFlagDescription = "LaunchDarkly access token with write-level access"
 	AnalyticsOptOutDescription = "Opt out of analytics tracking"
 	BaseURIFlagDescription     = "LaunchDarkly base URI"
+	DevStreamURIDescription    = "URI from which the dev server will stream flag configurations"
```

Comment:
In SDKs and docs we usually refer to this as "streaming endpoint" We could say something like:
> "URI for the dev server's flag streaming endpoint"

Reply:
This actually a different thing, this is the uri that the dev server uses to copy flags from, vs. an endpoint the dev server exposes. I don't think this description is particularly clear. Any thoughts on what it should say given that?

Reply:
Where it wires up the Go SDK to? 

> Streaming service endpoint that the CLI [or you could say LaunchDevly] uses to obtain authoritative flag data. This may be a LaunchDarkly or Relay Proxy endpoint.


I could see changing the description of `BaseURI` too to differentiate its usage from this new one.